Patterns/Base-URL.yaml

```diff
@@ -5,17 +5,25 @@ PortRegex: !simpleRegex
   def: (:\d{1,5})
 ExtractionRestrictionRegex: !simpleRegex
   def: (?<=\s|[\'""\(\[:]|^)
+UnicodeRestrictionRegex: !simpleRegex
```

Comment:
Why add this here?
It would be better if this is the configurable part. Or if the regexes are assembled on startup. Now there are basically 3 pairs of repeated regexes due to only a minor change.